.github/workflows/canary-integration-test.yml

```
@@ -642,7 +642,7 @@ jobs:
           tests/scripts/github-action-helper.sh create_LV_on_disk $BLOCK
 
       - name: deploy cluster
-        run: tests/scripts/github-action-helper.sh deploy_cluster lvm
+        run: tests/scripts/github-action-helper.sh deploy_all_resources lvm
```

Per previous suggestion, callers would need two calls:
```suggestion
        run: |
          tests/scripts/github-action-helper.sh deploy_cluster lvm
          tests/scripts/github-action-helper.sh deploy_all_additional_resources_on_cluster
```